cEP-0020.md

```diff
@@ -0,0 +1,418 @@
+# Newcomer metrics and Gamification
+
+| Metadata |                                                        |
+| -------- | ------------------------------------------------------ |
+| cEP      | 20                                                     |
+| Version  | 1.0                                                    |
+| Title    | Newcomer metrics and Gamification system               |
+| Authors  | Shrikrishna Singh <mailto:krishnasingh.ss30@gmail.com> |
+| Status   | Proposed                                               |
+| Type     | Process                                                |
+
+## Abstract
+
+This cEP describes how a gamification system will automate the process
+of becoming a `Developer` from a `Newcomer`, as a part of this
+[GSoC project](https://summerofcode.withgoogle.com/projects/#5892040252981248).
+
+For the implementation of this process, This project will incorporate a
+gamification system which will allow us to track newcomer's progress and
+assign some points to each of the activity they do. There will be different levels
+based on points threshold and a newcomer will also get some special privileges
+after the completion of each level. There will also be different badges on the
+basis of performed activities by the newcomers.
+
+So, every newcomer at coala will have to complete the processes involved in
+the gamification system in order to become a developer at coala or to get the
+**coala Developer** badge.
+
+## Background
+
+### Newcomer Experience
+
+**The problems we have:**
+
+- How to attract more newcomers
+  and how to cope with the increasing amounts of newcomers
```

Comment:
why "and" here?  why not use a new bullet point?

why all these 'how' questions that end with a full stop '.' ?